routing/src/prefix.rs

```
@@ -166,7 +167,7 @@ impl Prefix {
         T: TryInto<Prefix>,
         T::Error: Debug,
     {
-        val.try_into().expect("Invalid prefix")
+        val.try_into().unwrap()
```

why?

---

oh, I was debugging the test and the expect was eating the error message

---

Oh, I never realised it would :exploding_head: